grpc-server/src/main/scala/com/avast/sst/grpc/server/GrpcServerConfig.scala

```diff
@@ -0,0 +1,11 @@
+package com.avast.sst.grpc.server
+
+import java.util.concurrent.TimeUnit
+
+import scala.concurrent.duration.Duration
+
+final case class GrpcServerConfig(port: Int,
+                                  handshakeTimeout: Duration,
+                                  maxInboundMessageSize: Int = 4 * 1024 * 1024,
+                                  maxInboundMetadataSize: Int = 8192,
+                                  serverShutdownTimeout: Duration = Duration(10, TimeUnit.SECONDS))
```

Comment:
Wouldn't the `java.time.Duration` be more appropriate?

Reply:
It was decided the SST will use `scala.concurrent.duration.Duration` instead everywhere.

Reply:
Both options are possible but we've talked about it with some people and decided for the Scala `Duration`.

In the past I have read some comment that Scala's `Duration` was never supposed to be used as a general time/date data type so I was always preferring `java.time.Duration`. However in this case it's not really "work with time" but only a data type to transfer some duration value correctly and since this is mostly Scala library that will be interoperating with other Scala libraries I suppose choosing Scala's `Duration` is a better option.